new-project/src/components/UserInputFormExample/index.jsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
+
+export default function FormWithHook() {
+  const [name, setName] = useState("");
+  const [surname, setSurname] = useState("");
+  const [age, setAge] = useState("");
+  const [sex, setSex] = useState("");
```

Comment:
Try to merge these states into one, create an object with users' data and this object will change on some action